bot/src/main/kotlin/io/heapy/kotbot/bot/KotBot.kt

```diff
@@ -39,6 +47,26 @@ class KotBot(
                 execute(KickChatMember(action.chatId, action.userId))
                 Unit
             }
+            is SendMessageAction -> {
+                execute(SendMessage(action.chatId, action.text).also {
+                    if(action.inlineKeyboard != null) {
+                        it.replyMarkup = InlineKeyboardMarkup().apply { keyboard = action.inlineKeyboard }
+                    }
+                })
+                Unit
+            }
+        }
+    } catch (e: TelegramApiRequestException) {
+        val code = TelegramError.byCode(e.errorCode)
+        when {
```

Comment:
`when(val code = ...)`?

Reply:
I'm not only checking the `code`, but also `action` type, so that's not possible. It may be the case that in future not all `Action`s would be bound to chat.